lib/ansible/modules/apt.py

```diff
@@ -91,6 +91,16 @@
     type: bool
     default: 'no'
     version_added: "2.1"
+  allow_downgrade:
```

Comment:
Maybe change it to `allow_downgrades` to make it more similar to `allow-downgrades` option in apt itself?
```suggestion
  allow_downgrades:
```

Reply:
The reason why I used the singular form is for consistency with the option in the yum and dnf modules: https://docs.ansible.com/ansible/2.9/modules/yum_module.html
Happy to change it though if you think it's important.

Reply:
Ah, making it similar to other package modules is important too, I don't lean towards either side (apt itself, other package manager modules).